### PR TITLE
FOIA-243: Inline documentation and README for foia_annual_report_data module.

### PR DESCRIPTION
**Pull request by jesseday**
* Makes changes requested in this PR: https://github.com/acquia-pso/foia-api-enhancements/pull/325
* The original branch for this work is https://github.com/alozie/foia-api-enhancements/tree/FOIA-243-annual-report-doc

### CODE REVIEW

File: docroot/modules/custom/foia_annual_data_report/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+No configuration required for the node edit form updates.
+
+To configure the Annual Report Memory limit go to:
+/admin/config/system/foia_annual_data_report_memory_limit
+
+ * Fill in the `Annual Report Memory Limit` field  if you wish to override the
+default PHP Memory Limit set for the platform. The value set in the `Annual
+Report Memory Limit` field must be in the form of a numeric value and a
+capitalized M as the unit symbol, with no space between the two, e.g. `1024M`.
+ * Check the `Debug Annual Report memory limit.` checkbox to log the memory
```
Comment (bemarlan) [inline docroot/modules/custom/foia_annual_data_report/README.md]: > Check the `Debug Annual Report memory limit.`

Place back tick before the period.

File: docroot/modules/custom/foia_annual_data_report/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+the codebase is not installed via composer. Enable as you would normally
+enable a contributed Drupal module. Visit
+https://www.drupal.org/docs/8/extending-drupal-8/installing-drupal-8-modules#s-step-2-enable-the-module
+for further information.
+
+
+CONFIGURATION
+-------------
+
+No configuration required for the node edit form updates.
```
Comment (bemarlan) [inline docroot/modules/custom/foia_annual_data_report/README.md]: A user needs permission 'administer site configuration' in order to change config for this module. We should probably note that here.

File: docroot/modules/custom/foia_annual_data_report/README.md (hunk 1 above)
Comment (jesseday) [reply docroot/modules/custom/foia_annual_data_report/README.md]: The period had actually been part of the field label, but it's a little odd.  Removed the period from the field label and the readme.

File: docroot/modules/custom/foia_annual_data_report/README.md (hunk 2 above)
Comment (jesseday) [reply docroot/modules/custom/foia_annual_data_report/README.md]: Documented

**Pull request merged**